celements-user-manager/web-module/src/main/webapp/resources/celJS/userAdmin/sendActivationMail.mjs

```diff
@@ -0,0 +1,44 @@
+class SendMailButton {
+  #htmlElemA;
+  #iconElem;
+  constructor(htmlElemA) {
+    this.#htmlElemA = htmlElemA;
+    this.#iconElem = this.#htmlElemA.querySelector(".icon");
+  }
+
+  register() {
+    console.log('registering event listener for ' + this.#htmlElemA);
+    this.#htmlElemA.addEventListener('click', (ev) => this.#stopDefaultAction(ev));
+  }
+
+  #stopDefaultAction(event) {
```

Comment:
this is doing more than "stopDefaultAction", maybe named it `sendActivationMailHandler` or `handleEventForSendActivationMail`